scss/tech.scss

```diff
@@ -0,0 +1,59 @@
+@import "colors";
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$tech-logo-size: 100px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+
+.boxes {
+  display: flex;
+  flex-flow: row;
+  flex-wrap: wrap;
+  padding: 0px 0px 20px 100px;
```

Comment:
the 20px at the bottom should be added to the margin since we want to separate the boxes from the following title. the 100px at the left is ok but not adding the same value at the right makes it not centered, id use `padding: 0 100px` instead and a margin bottom of 70px

Reply:
also when the value is 0 don't add the `px`